service/src/main/java/uk/gov/hmcts/reform/fpl/service/payment/PaymentService.java

```diff
@@ -108,15 +110,20 @@ private CreditAccountPaymentRequest getCreditAccountPaymentRequest(Long caseId,
             .build();
     }
 
-    private void callPaymentsApi(CreditAccountPaymentRequest creditAccountPaymentRequest) {
+    @Retryable(value = PaymentRetryException.class)
+    //had to make this public for retry to work
+    public void callPaymentsApi(CreditAccountPaymentRequest creditAccountPaymentRequest) {
         try {
             paymentApi.createCreditAccountPayment(requestData.authorisation(),
                 authTokenGenerator.generate(),
                 creditAccountPaymentRequest);
+        } catch (FeignException.InternalServerError ex) {
+            log.error("Internal server error caught, payment will be retried");
```

Comment:
do we want to log it as en error if we want to retry?

Reply:
done